scripts/run-integration-tests.sh

```diff
@@ -216,11 +224,12 @@ echo "**************************************************************************
 echo "Running integration tests on default CNI version, $ADDONS_CNI_IMAGE"
 echo ""
 START=$SECONDS
-pushd ./test/integration
-GO111MODULE=on go test -v -timeout 0 ./... --kubeconfig=$KUBECONFIG --ginkgo.focus="\[cni-integration\]" --ginkgo.skip="\[Disruptive\]" \
-    --assets=./assets
+
+focus="CANARY"
```

Comment:
So for now you are just enabling canary tests?

Reply:
Yes, since there is some flakyness when running all suites at once. We will incrementally expand this. Also this is in alignment to what we have running on prow infra.

Reply:
But this will run for each PR merge and nightly runs so I agree we can do it incrementally but can we take it up as a fast follow up to enable all tests? Since canary tests doesn't cover all scenarios.